internal/operator/clusterutilities.go

```diff
@@ -778,20 +778,36 @@ func GetPgbackrestS3EnvVars(cluster crv1.Pgcluster, clientset kubernetes.Interfa
 	} else {
 		s3EnvVars.PgbackrestS3URIStyle = Pgo.Cluster.BackrestS3URIStyle
 	}
-	if cluster.Spec.BackrestS3VerifyTLS != "" {
-		s3EnvVars.PgbackrestS3VerifyTLS = cluster.Spec.BackrestS3VerifyTLS
-	} else {
-		s3EnvVars.PgbackrestS3VerifyTLS = Pgo.Cluster.BackrestS3VerifyTLS
-	}
 
+	// if the URI style is not configured, set to the default value
+	if s3EnvVars.PgbackrestS3URIStyle == "" {
+		s3EnvVars.PgbackrestS3URIStyle = "host"
+	}
 	// if set, pgBackRest URI style must be set to either 'path' or 'host'. If it is neither,
 	// log an error and stop the cluster from being created.
-	if s3EnvVars.PgbackrestS3URIStyle != "path" && s3EnvVars.PgbackrestS3URIStyle != "host" &&
-		s3EnvVars.PgbackrestS3URIStyle != "" {
+	if s3EnvVars.PgbackrestS3URIStyle != "path" && s3EnvVars.PgbackrestS3URIStyle != "host" {
 		log.Error("pgBackRest S3 URI style must be set to either \"path\" or \"host\".")
 		return ""
 	}
 
+	// get the verify TLS boolean value as a string
+	s3EnvVars.PgbackrestS3VerifyTLS = GetS3VerifyTLSSetting(&cluster)
+
+	doc := bytes.Buffer{}
+
+	if err := config.PgbackrestS3EnvVarsTemplate.Execute(&doc, s3EnvVars); err != nil {
+		log.Error(err.Error())
+		return ""
+	}
+
+	return doc.String()
+}
+
+// GetS3VerifyTLSSetting parses the configured value as a boolean to ensure a valid
+// option is used, then returns the pgBackRest S3 configuration value to either enable
+// or disable TLS verification as the expected string value.
+func GetS3VerifyTLSSetting(cluster *crv1.Pgcluster) string {
```

Comment:
I believe very call to this passes in a pointer reference, so unsure why we are having this accept a pointer and not just a `crv1.Pgcluster` itself.